data/sql/updates/pending_db_world/rev_1611875417271883500.sql

```diff
@@ -1,9 +1,12 @@
 INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611875417271883500');
 
+/*Cleanup GUIDs that do not exist - These GUIDs were migrated to GUID 241000 and 241002 -241006*/
+DELETE FROM `game_event_gameobject` WHERE `eventEntry` = 7 and `guid` IN (11995,11996,11997,11998,11999,12000); 
```

Comment:
but they exist for me. Is it intended to delete them in this PR?

![image](https://user-images.githubusercontent.com/75517/106354552-00819780-62f3-11eb-9000-f5954a460758.png)

Reply:
No shin. Check gameobject instead. This table is not relevant if the GUID does not exist in gameobject

Reply:
ah right